assets/js/RepLog/RepLogs.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import RepLogList from './RepLogList';
+import RepLogCreator from './RepLogCreator';
+//import RepLogCreator from './RepLogCreatorControlledComponents';
+
+function calculateTotalWeightLifted(repLogs) {
+    let total = 0;
+
+    for (let repLog of repLogs) {
+        total += repLog.totalWeightLifted;
+    }
+
+    return total;
+}
```

Comment:
Alternative:

```js
const totalWeigth = replogs => replogs.reduce((total, log) => total + log.totalWeightLifted, 0);
```

Which is so small it could even be inlined.